app/main.py

```
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
```

Consider providing feedback or raising an exception if the command format is incorrect. Currently, the function silently returns without any indication of what went wrong.